### PR TITLE
Fix the case of some views still rendered twice on the client

### PR DESCRIPTION
**Pull request by ingiulio**
See #461

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ coverage
 *.sw?
 .DS_Store
 runtime.json
+.idea
```
Comment (ethan-trulia) [inline .gitignore]: a jetbrains file?

Comment (bigethan) [inline .gitignore]: a jetbrains file?

**Comment by bigethan** [comment]
:+1:

**Comment by purusho** [comment]
:+1:

File: .gitignore (hunk 1 above)
Comment (ingiulio) [inline .gitignore]: yep, it's a hidden folder where jetbrains IDE generate all sort of project files

Comment (bigethan) [inline .gitignore]: should it be in this PR?

Comment (ingiulio) [inline .gitignore]: I think it comes handy if one is using one of these IDEs. Would you rather see it in a separate PR or not at all? :)

Comment (bigethan) [inline .gitignore]: it's a slippery slope of adding all files ever? I'd prefer if this was added to your `~/.gitignore` which you can customize with whatever you personally are running. (enable this via `git config --global core.excludesfile ~/.gitignore`)

**Comment by bigethan** [comment]
@saponifi3d any thoughts on this PR? Have you run into this issue as well?

File: .gitignore (hunk 1 above)
Comment (ingiulio) [inline .gitignore]: Makes sense. I didn't know about this option. Thanks, I'll update!

**Comment by alexindigo** [comment]
@rendrjs/maintainers Is there anything else we need to do to land this PR? Thank you.

**Comment by saponifi3d** [comment]
Hey all - sorry for the late response.  Can you explain the context that you were seeing that a view was being rendered twice? Thanks!

**Comment by saponifi3d** [comment]
Looks good to me :dancers:

**Comment by ingiulio** [comment]
thank you @saponifi3d!

**Pull request merged**